spec/lib/miq_automation_engine/miq_ae_discovery_spec.rb

```diff
@@ -4,12 +4,15 @@ module MiqAeDiscoverySpec
   include MiqAeEngine
```

Comment:
@lfu 
After you run this spec can you look at the Automation log, it has errors in it. Not sure if they are legacy stuff and we should remove them.